vue/components/ui/organisms/input-symbol/input-symbol.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div
+        class="input-symbol"
+        v-bind:class="[{ disabled: disabled }, `color-${variant}`, `border-${border}`]"
+    >
+        <input-ripe
+            class="input-ripe"
+            v-bind:value.sync="valueData"
+            v-bind:variant="variant"
+            v-bind:border="border"
+            v-bind:placeholder="placeholder"
+            v-bind:disabled="disabled"
+            v-bind:width="width"
+            v-bind:height="height"
+            v-on:update:value="$emit('update:value', $event)"
+            v-on:blur="$emit('blur')"
+            v-on:focus="$emit('focus')"
+        />
+        <div class="symbol" v-bind:style="style">
+            {{ symbol }}
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.input-symbol.disabled {
+    opacity: 0.4;
+}
+
+.input-symbol .input-ripe {
+    border-radius: 6px 0px 0px 6px;
+    padding-right: 7px;
+    text-align: right;
+    text-overflow: ellipsis;
+    vertical-align: top;

```

Comment:
Does this do anything?

Reply:
Yes, aligns `input-symbol` in the same horizontal level as `input-ripe`
<img width="289" alt="imagem" src="https://user-images.githubusercontent.com/24736423/72250927-ab45eb80-35f4-11ea-98f9-37eb7a96f314.png">